crates/bevy_input/src/keyboard.rs

```diff
@@ -60,7 +61,7 @@ pub fn keyboard_input_system(
 /// ## Updating
 ///
 /// The resource is updated inside of the [`keyboard_input_system`](crate::keyboard::keyboard_input_system).
-#[derive(Debug, Hash, Ord, PartialOrd, PartialEq, Eq, Clone, Copy)]
+#[derive(Reflect, FromReflect, Debug, Hash, Ord, PartialOrd, PartialEq, Eq, Clone, Copy)]
```

Comment:
Any reason to not register reflected traits like `PartialEq` and `Hash`?

(Also on `ScanCode`)

Reply:
@jakobhellermann, I'd like to address this (either way) before I merge. This is the only remaining blocker IMO.